packages/front-end/components/Settings/MemberList.tsx

```diff
@@ -72,28 +129,40 @@ const MemberList: FC<{
               <td>
                 {member.id !== userId && (
                   <>
-                    <a
-                      href="#"
-                      className="tr-hover mr-3"
-                      onClick={(e) => {
-                        e.preventDefault();
-                        setRoleModal(member);
-                        setRole(member.role);
-                      }}
-                    >
-                      <FaPencilAlt />
-                    </a>
-                    <DeleteButton
-                      link={true}
-                      className="tr-hover"
-                      displayName={member.email}
-                      onClick={async () => {
-                        await apiCall(`/member/${member.id}`, {
-                          method: "DELETE",
-                        });
-                        mutate();
-                      }}
-                    />
+                    <MoreMenu id="test">
+                      <button
+                        className="dropdown-item pe-auto"
+                        onClick={(e) => {
+                          e.preventDefault();
+                          setRoleModal(member);
+                          setRole(member.role);
+                        }}
+                      >
+                        Edit Role
+                      </button>
+                      <button
```

Comment:
We only want to render this button on self-hosted installations.  There's an `isCloud` helper function on the front-end for this.

Reply:
@jdorn In my testing on localhost, `isCloud` is returning true. Is that correct? Am I having a brain fart? I would've expected it to be false.

Reply:
`isCloud` is a function, so doing `if (isCloud)` is always true, but `if (isCloud())` will work as expected.  I think there's a way to make typescript catch these types of typos since they are really common and extremely frustrating to track down once they are in production.

Reply:
Ah - my mistake. Thank you!